x/wasm/internal/keeper/test_common.go

```diff
@@ -388,12 +405,12 @@ func handleExecute(ctx sdk.Context, k *Keeper, msg *types.MsgExecuteContract) (*
 	return res, nil
 }
 
-func RandomAccountAddress(_ *testing.T) sdk.AccAddress {
+func RandomAccountAddress(_ TestingT) sdk.AccAddress {
```

Comment:
There is a `testing.TB` interface I have seen before that probably is what you want here - one interface that lets you use the code both in unit tests as well as benchmarks.